include/alpaka/core/ApiCudaRt.hpp

```diff
@@ -103,67 +103,67 @@ namespace alpaka
         };
 
         // Runtime API
-        static inline Error_t deviceGetAttribute(int* value, DeviceAttr_t attr, int device)
+        inline static Error_t deviceGetAttribute(int* value, DeviceAttr_t attr, int device)
```

Comment:
Intuitively, I prefer `static inline`, because `static` is the more important information here for me. Maybe we should rethink the clang-format option here.

Reply:
I will change this in the contributing repository.